scheduled-jobs/build/sync-operator-sdk/Jenkinsfile

```diff
@@ -0,0 +1,29 @@
+node {
+    checkout scm
+    buildlib = load("pipeline-scripts/buildlib.groovy")
+    commonlib = buildlib.commonlib
+
+    properties( [
+        buildDiscarder(logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '100', daysToKeepStr: '', numToKeepStr: '100')),
+        disableConcurrentBuilds(),
+        disableResume(),
+    ] )
+
+    for ( version in commonlib.ocp4Versions ) {
+        if (!commonlib.is_prerelease(version)) { // Skip pre-release
+            def cv = "${version}" // make sure we use a locally scoped variable
+
+            // Skip the build if already locked
+            activityLockName = "github-activity-lock-${cv}"
```

Comment:
I didn't see a reason to hold `github-activity-lock`. This kind of locks are mainly to prevent parallel image/rpm builds at this moment. It is better to remove this lock or use a different lock if you think it is necessary.